MDANSE/Src/MDANSE/Trajectory/H5MDTrajectory.py

```diff
@@ -497,16 +497,10 @@ def read_atomic_trajectory(
 
         if self._unit_cells is not None:
             direct_cells = np.array(
-                [
-                    self.unit_cell(nf).transposed_direct
-                    for nf in range(first, last, step)
-                ]
+                [self.unit_cell(nf).direct for nf in range(first, last, step)]
             )
             inverse_cells = np.array(
-                [
-                    self.unit_cell(nf).transposed_inverse
-                    for nf in range(first, last, step)
-                ]
+                [self.unit_cell(nf).inverse for nf in range(first, last, step)]
             )
             atomic_traj = atomic_trajectory(
                 coords, direct_cells, inverse_cells, box_coordinates
```

Comment:
I think we also need to update atomic_trajectory since that function expects the transpose versions of direct and inverse.